.gitignore

```diff
@@ -65,6 +65,8 @@ install/kubernetes/istio-init.yaml
 install/kubernetes/istio-non-mcp.yaml
 install/kubernetes/istio-minimal.yaml
 install/kubernetes/helm/istio/requirements.lock
+manifests/charts/istiod-remote/files/injection-template.yaml
```

Comment:
How are these used if they are in gitignore? They won't be published anywhere then, is that intended?

Reply:
basically these files are generated as part of make gen.  Having them here will ensure we don't accidentally check them in git.  remember you requested them not in git!

Reply:
How are you going to use it though? I assume some user would download the istio tar.gz and run `helm install charts/istiod-remote` but these files will be missing?

Reply:
oh :) that is a good point.  Wouldn't the istio tar.gz include the `make gen` artifact?   Perhaps that is my wishful thinking.

Reply:
ok I will check them in.  It will work when user clones the repo as well without running `make gen`